docs/references/qa/CometBFT-QA-v1.md

```diff
@@ -0,0 +1,302 @@
+
+| v0.38 | v1 (without LE / with LE) 
+| :--------------:|:--------------:|
+| ![mempool-avg-baseline](img38/200nodes/avg_mempool_size.png) | ![mempool-avg](imgs/v1/200nodes/metrics/avg_mempool_size.png)
+| | ![mempool-avg-le](imgs/v1/200nodes_with_latency_emulation/metrics/avg_mempool_size.png)
+
+**Maximum size** The maximum mempool size indicates when one or more nodes have reached their
+maximum capacity in terms of the number of transactions they can hold. In version `v0.38`, it's
+apparent that most of the time, at least one node that is dropping incoming transactions.
+Conversely, in `v1`, this happens less often, especially after reaching round 1 (as detailed below).
+
+However, when we introduce latency emulation into `v1`, there is consistently at least one node with
+a saturated mempool.
+
+| v0.38 | v1 (without LE / with LE)
+| :--------------:|:--------------:|
+| ![mempool-cumulative-baseline](img38/200nodes/mempool_size_max.png) | ![mempoool-cumulative](imgs/v1/200nodes/metrics/mempool_size_max.png)
+| | ![mempoool-cumulative-le](imgs/v1/200nodes_with_latency_emulation/metrics/mempool_size_max.png)
+
+#### Peers
+
+On all expertiments, the number of peers was stable on all nodes. As expected, the seed nodes have
+more peers (around 125) than the rest (between 20 and 70 for most nodes). The red dashed line
+denotes the average value. Just as in the baseline, the fact that non-seed nodes reach more than 50
+peers is due to [\#9548].
+
+| v0.38 | v1 (without LE / with LE) 
+|:--------------:|:--------------:|
+| ![peers](img38/200nodes/peers.png) | ![peers](imgs/v1/200nodes/metrics/peers.png)
+| | ![peers](imgs/v1/200nodes_with_latency_emulation/metrics/peers.png)
+
+#### Consensus rounds
+
+On both versions, most blocks took just one round to reach consensus, except for a few cases when it
+was needed a second round. On these two particular runs, we observe that `v0.38` required an extra
+round on more occasions than `v1`.
+
+With latency emulation, the performance is notably worse: the consensus module requires an extra
+round more often, even needing four rounds to finalise a block.
+
+| v0.38 | v1 (without LE / with LE) 
+|:--------------:|:--------------:|
+| ![rounds](img38/200nodes/rounds.png) | ![rounds](imgs/v1/200nodes/metrics/rounds.png)
+| | ![rounds](imgs/v1/200nodes_with_latency_emulation/metrics/rounds.png)
+
+#### Blocks produced per minute and transactions processed per minute
+
+These figures show the rate in which blocks were created from the point of view of each node,
+indicating when each node learned that a new block had been agreed upon. Throughout much of the load
+application period, the majority of nodes maintained a rate of approximately 20 blocks per minute. A
+spike to more than 100 blocks per minute occurred due to a slower node catching up. This behavior was similar in the baseline scenario.
+
+With latency emulation (LE), there is a noticeable degradation in performance. The block generation
```

Comment:
But it seems that throughput is not affected with LE -- or throughput is positively affected.

Since there's fewer blocks / minute with LE, this should mean that blocks tend to be bigger (more full) with transactions. 

Do we have the data to confirm this?

Reply:
Yes, I added new images with metrics of block size in Mb.